tile-group.lisp

```diff
@@ -31,6 +31,12 @@
    (last-frame :initform nil :accessor tile-group-last-frame)
    (current-frame :accessor tile-group-current-frame)))
 
+(defmethod tile-group-current-frame :before ((group tile-group))
+  (declare (optimize (speed 0) (debug 3) (safety 3)))
```

Comment:
Protip: For debugging purposes you can add `(restrict-compiler-policy 'debug 3)` to your `.sbclrc` so you don't have to be specific as to which function to you to ensure to have been compiled with a high debug setting.

Reply:
thanks I've missed those two declarations. removed now in https://github.com/m-g-r/stumpwm/pull/1